cmd/ygg/main.go

```diff
@@ -154,8 +154,58 @@ func main() {
 		{
 			Name:  "status",
 			Usage: "reports status of connection and activation",
+			Flags: []cli.Flag{
+				// TODO: load this value from config ? (or have a default)
+				&cli.StringSliceFlag{
+					Name: "broker",
+				},
+			},
 			Action: func(c *cli.Context) error {
-				return fmt.Errorf("not implemented")
+				hostname, err := os.Hostname()
+				if err != nil {
+					// default for messaging if hostname can't be read
+					hostname = "host"
+				}
+				fmt.Printf("Connection status for %s:\n\n", hostname)
+
+				// get RHSM registration status
+				uuid, err := getConsumerUUID()
+				if err != nil {
+					fmt.Printf("⛔️ ERROR: Unable to check Red Hat Subscription Manager status: %s\n\n", err)
+				} else {
+
```

Comment:
Did you run this through `go fmt`? I thought it removed blank lines in places like this.